homeassistant/components/camera/amcrest.py

```diff
@@ -0,0 +1,140 @@
+"""
+Support for Amcrest cameras.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.amcrest/
+"""
+import logging
+
+import requests
+
+from homeassistant.components.camera import DOMAIN, Camera
+from homeassistant.helpers import validate_config
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=too-many-instance-attributes
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Discover cameras on a Unifi NVR."""
+    if not validate_config({DOMAIN: config},
```

Comment:
Please don't use validate_config but `voluptuous` instead to create a `PLATFORM_SCHEMA`. [See this example.](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/lg_netcast.py#L36-L42)